files/justfiles/hardening.just

```diff
@@ -106,13 +106,14 @@ flatpak-permissions-lockdown:
 setup-usbguard:
     #!/usr/bin/bash
     echo "Notice: This will generate a policy based on your existing connected USB devices."
-    ACTIVE_USERNAME=$(whoami)
     run0 sh -c '
         mkdir -p /var/log/usbguard
         mkdir -p /etc/usbguard
         chmod 755 /etc/usbguard
         usbguard generate-policy > /etc/usbguard/rules.conf
         systemctl enable --now usbguard.service
         usbguard add-user $1
-    ' -- $ACTIVE_USERNAME
+	sed -i "/IPCAllowedGroups=wheel/s/$/ ipc/" /etc/usbguard/usbguard-daemon.conf
```

Comment:
Is this the standard/recommended group to use for usbguard access? Why not make a new group called `usbguard`?

Reply:
There wasn't really a standard/recommended group afaict, but the reason I chose it is because of the error you get if you don't have it:
`IPC ERROR: request id=1: IPC method: usbguard.IPC.x: Permission denied`
As well as the actual name of the setting:
`IPCAllowedGroups`

Reply:
A `usbguard` group would work exactly the same if that's how you'd prefer it

Reply:
@radioaddition ah I see, I think I'm misunderstanding something. Is ipc a group name you chose arbitrarily or is it used for other things?

Reply:
It's a group name I chose arbitrarily

Reply:
ah okay, might as well make it "usbguard" then ¯\_(ツ)_/¯

Reply:
Will do